packages/forma-36-react-components/src/components/Asset/README.mdx

```diff
@@ -0,0 +1,107 @@
+import { Asset } from './Asset';
+
+# Asset
+
+[Source code](https://github.com/contentful/forma-36/blob/master/packages/forma-36-react-components/src/components/Asset/Asset.tsx)
+
+The asset component is a visual representation of an asset file such as a video or a PDF document.
+
+<Asset title="Archive of files" type="archive" />
+
+## Table of contents
+
+- [Usage](#usage)
+  - [With an image](#with-an-image)
+  - [With link and target](#with-link-and-target)
+- [Props](#props)
```

Comment:
I think this Table of contents is a leftover from the Card MDX, isn't it? hahaha